cmd/support_archive/eec_diag_logs.go

```diff
@@ -0,0 +1,292 @@
+package support_archive
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/logd"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/installconfig"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/labels"
+	"github.com/pkg/errors"
+	"golang.org/x/net/context"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	apilabels "k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/kubernetes/scheme"
+	clientgocorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/remotecommand"
+	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
+)
+
+const (
+	diagLogCollectorName = "diagLogCollector"
+	eecExtensionsPath    = "/var/lib/dynatrace/remotepluginmodule/log/extensions"
+	fileNotFoundMarker   = "<NOT FOUND>"
+	EecPodName           = "dynatrace-extensions-controller"
+	eecContainerName     = "extensions-controller"
+)
+
+type executionResult struct {
+	StdOut *bytes.Buffer
+	StdErr *bytes.Buffer
+}
+
+type diagLogCollector struct {
```

Comment:
What does "diag" stand for?

If it's "diagnostic", then are there "none-diagnostic" logs 😅

Reply:
It could be "fsLogCollector"
Because it's getting them from the filesystem

Reply:
Or "execLogCollector" because it uses exec to get the logs

Reply:
good point! please rename it

Reply:
right, all logs are more or less diagnostic logs. Renamed to `fsLogCollector`